web-jehlomat/src/screens/Nalezy/Components/Map.tsx

```diff
@@ -1,103 +1,26 @@
 import React, { FunctionComponent, useEffect } from 'react';
-import { MapContainer, Marker, Popup, TileLayer, useMap } from 'react-leaflet';
-import L, { Icon, LatLngTuple } from 'leaflet';
-import { styled } from '@mui/system';
+import { Marker, TileLayer, useMap } from 'react-leaflet';
+import L, { LatLngTuple } from 'leaflet';
 import { DEFAULT_POSITION, DEFAULT_ZOOM_LEVEL } from 'screens/NovyNalez/constants';
 import { Syringe } from 'screens/Nalezy/types/Syringe';
 import { SyringeReadModel } from 'screens/Nalezy/types/SyringeReadModel';
-import { SyringeState } from 'screens/Nalezy/types/SyringeState';
 import { Loader } from 'utils/Loader';
 import Loading from 'screens/Nalezy/Components/Loading';
 import PreviewSyringeState from 'screens/Nalezy/Components/SyringeState';
 import Links from 'screens/Nalezy/Components/Links';
+import LeafletMap from 'screens/Nalezy/Components/LeafletMap';
+import { PinMenu, Info, Location, Time, State } from 'screens/Nalezy/Components/PinMenu';
 
 import 'leaflet/dist/leaflet.css';
-import gray from 'assets/pins/pin-gray.svg';
-import green from 'assets/pins/pin-green.svg';
-import yellow from 'assets/pins/pin-yellow.svg';
-import dayjs from "dayjs";
+
+import dayjs from 'dayjs';
```

Comment:
library inport on top please ...

Reply:
I know, there is no coincidence that rest of my components follows this "rule". However this "rules" should be defined, preferably as a ESLint/Prettier rules or at least in the Contribution file.
Demand to following this opinioned "rules" is imho quite unfair without previous agreement.

Reply:
I am not demanding anything, I am just pointing out that it's standart practise. Same as other comments, it's completely up to you if you feel like it should be implemented.